tests/app/questionnaire/test_placeholder_renderer.py

```diff
@@ -3,12 +3,108 @@
     PlaceholderRenderer,
     find_pointers_containing,
 )
+from app.questionnaire.questionnaire_schema import QuestionnaireSchema
 from tests.app.app_context_test_case import AppContextTestCase
 
 
 class TestPlaceholderRenderer(AppContextTestCase):
     def setUp(self):
         super().setUp()
+        self.default_question_schema = QuestionnaireSchema(
```

Comment:
This doesn't need to appear in the setup, given it's only used in a single test. Probably worth putting alongside the other placeholder schemas as test_placeholder_defaults or something, given theres value in being able to test it manually.